domains/aniruddh.is-an.app.json

```
@@ -0,0 +1,14 @@
+{
+	"$schema": "../schemas/domain.schema.json",
+	"description": "My personal project",
```

Could you please provide a description (what is it) here? Is your pet project or something else?
```suggestion
	"description": "write-it-hete",
```

---

done